app/livechat/client/views/app/integrations/livechatIntegrationWebhook.js

```diff
@@ -73,12 +76,12 @@ Template.livechatIntegrationWebhook.events({
 	'click .reset-settings'(e, instance) {
 		e.preventDefault();
 
-		const webhookUrl = LivechatIntegration.findOne('Livechat_webhookUrl');
-		const secretToken = LivechatIntegration.findOne('Livechat_secret_token');
-		const webhookOnClose = LivechatIntegration.findOne('Livechat_webhook_on_close');
-		const webhookOnOfflineMsg = LivechatIntegration.findOne('Livechat_webhook_on_offline_msg');
-		const webhookOnVisitorMessage = LivechatIntegration.findOne('Livechat_webhook_on_visitor_message');
-		const webhookOnAgentMessage = LivechatIntegration.findOne('Livechat_webhook_on_agent_message');
+		const webhookUrl = getIntegrationSettingById(instance.settings.get(), 'Livechat_webhookUrl');
```

Comment:
Since you get the original data from the `instance.settings` to reset all settings, you need to update the `instance.settings` after you submit the data to the server, otherwise, if the user clicks on the `reset` button, then the older data will be set to the inputs.